server/models/Article.js

```diff
@@ -1,9 +1,21 @@
 const mongoose = require('mongoose');
 const { Schema } = mongoose;
+const CategoryModel = require('../models/Category');
 
 const schema = new Schema({
     title: String,
-    fullContent: String
+    fullContent: String,
+    postedBy: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: CategoryModel
+    },
+    categoryName: [{
```

Comment:
this should be called called `categoryId` or `category` and be of type reference to CategoryModel